src/System.Management.Automation/DscSupport/CimDSCParser.cs

```diff
@@ -3221,6 +3283,21 @@ public static bool ImportCimKeywordsFromModule(PSModuleInfo module, string resou
 
             return false;
         }
+        
+        /// <summary>
+        /// Clear the 'IsImportedImplicitly' flag when explicitly importing a resource
+        /// </summary>
+        /// <param name="module"></param>
+        /// <param name="cimClass"></param>
+        public static void ClearImplicitlyImportedFlagFromResourceInClassCache(PSModuleInfo module, CimClass cimClass)
```

Comment:
This is a public method on an internal class so it is not public to every one?

Reply:
I will make it private, its not being used outside of this class.

---
In reply to: [293527097](https://github.com/PowerShell/PowerShell/pull/9879#discussion_r293527097) [](ancestors = 293527097)